[loganalyzer] implementation with sample files

hrachya-mellanox opened the pull request:
* loganalyzer.py - log analyzer to analyze system log file and
additionally specified input files.

signed-off-by: hrachya@mellanox.com

andriymoroz commented on ansible/roles/test/files/tools/loganalyzer.py:
```diff
@@ -0,0 +1,578 @@
+    return ret_code
+#---------------------------------------------------------------------
+
+def check_logs(logs):
```
looks like this function is never used...

stcheng commented:
could you update this pull request to compare it with the current master?

taoyl-ms commented:
Could we have more document/comment on the meaning and usage of those sample files?

hrachya-mellanox commented:
Hi Taoyu,
The usage of files is described in https://github.com/Azure/SONiC/wiki/LogAnalyzer#analysis-of-log-messages

taoyl-ms commented:
I was talking about the log files in samples/sample1 folder. Shall we modify the sample string and runID there to be more meaningful? I'm feeling current sample difficult to follow

hrachya-mellanox commented:
Hi Taoyu,
Following the description in the wiki page it should be clear I think, but if you'd like to suggest different content, you can send through mail and I can add it.

taoyl-ms commented:
Hi Hrachyam,

What I mean is that we might be able to do some modification in the details to make the sample easier to read. Such as
-	In command_line.txt, modify the file paths to relative path in repo instead of /home/hrachya/projects/… so that user knows that those files can be found in the reop
-	Rename command_line.txt to README.txt or other obvious name so readers know this is the first file they need to read
-	Rename runID to something meaningful instead of myTest116
-	Write some descriptive message in ignore string sample instead of random strings like “67hello8"

hrachya-mellanox commented:
i removed the sample directory, added sample to https://github.com/Azure/SONiC/wiki/LogAnalyzer. I'm also going to open new pull request for ansible files invoking loganalyzer which should help understanding its usage better.
